### DISCUSSION TITLE
Approved but GitHub Education benefits not available

### DISCUSSION THREAD

**Discussion by prajyotdeshpande23** [community/community#135849] [GitHub Education] [answered]
### Hi everyone,

I got approved on 7th August, but I still have not received my benefits yet.
Please look into it and help me. 
![Screenshot 2024-08-15 121003](https://github.com/user-attachments/assets/69f090ea-3ce3-4d0e-9774-a5b5a6ff5163)

**Comment by seamoonpandey** [comment]
It takes few days, all you need is patience. Once the benefits are available the Green tag of approved turns to purple

**Comment by davevad93** [comment, marked as answer]
Hi @prajyotdeshpande23 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @prajyotdeshpande23 was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by prajyotdeshpande23** [reply]
Thank You @davevad93 !

**Reply by davevad93** [reply]
You're welcome @prajyotdeshpande23 ! 👍
